application/accountmanager/src/accountmanager/main.py

```
@@ -0,0 +1,30 @@
+async def get_health():
+    return {"status": "healthy"}
+
+
+@application.get("/account")
+async def get_account():
+    try:
+        client = get_client()
+        account = client.get_account()
+        return account.model_dump()
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)